waku/node/waku_node.nim

```diff
@@ -94,6 +95,7 @@ type
     wakuLegacyStoreClient*: legacy_store_client.WakuStoreClient
     wakuStore*: store.WakuStore
     wakuStoreClient*: store_client.WakuStoreClient
+    wakuStoreResume*: StoreResume
```

Comment:
Unless I'm missing something I think we should avoid that. Instead, I think the best place is to move it to the `Waku` instance.

Reply:
The logic will be expanded to include calling a Store client query in a callback. Store Client, Archive and Store Resume need to be tied together in `waku_node`.